src/App.js

```diff
@@ -1,18 +1,38 @@
 import './App.css';
+import './style/style.css';
+import './style/quotes.css';
 import React from 'react';
-import Calculator from './components/Calculator';
+import { Link, Route, Routes } from 'react-router-dom';
 import './components/logic/calculate';
 import './components/logic/operate';
+import Section from './components/Section';
+import Home from './components/Home';
+import Quotes from './components/Quotes';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
+// eslint-disable-next-line react/prefer-stateless-function
+function App() {
+  return (
+    <>
+      <div className="nav-bar">
+        <div className="navs">
+          <h1>Math Magicians</h1>
+          <div className="links">
+            <Link to="/Home"> Home </Link>
+            <Link to="/Quotes" className="qu">
+              Quotation
+            </Link>
+            <Link to="/Calculator"> Calculator </Link>
+          </div>
+        </div>
 
-  render() {
-    return <Calculator />;
-  }
+        <Routes>
+          <Route path="/Home" element={<Home />} />
```

Comment:
- [ ] Why dont you set this path to the home page so that when the app is loaded it will direct to the home page?
You can change this to :point_down: 
`<Route path="/" element={<Home />} />`